test-suite/convertiblebonds.cpp

```diff
@@ -78,10 +78,15 @@ struct CommonVars {
         frequency = Annual;
         settlementDays = 3;
 
-        issueDate = calendar.advance(today,2,Days);
+        fixingDays = 2;
+        issueDate = calendar.advance(today, fixingDays, Days);
         maturityDate = calendar.advance(issueDate, 10, Years);
         // reset to avoid inconsistencies as the schedule is backwards
         issueDate = calendar.advance(maturityDate, -10, Years);
+        fixingDays = 0;
+        for (Date d = today; d != issueDate; ++d)
+            if (calendar.isBusinessDay(d) && !calendar.isWeekend(d.weekday()))
+                ++fixingDays;
```

Comment:
I suppose there is already some functionality implemented for this, but I did not find it. 
Note, `Calendar::businessDaysBetween()` also includes weekends.

Reply:
`Calendar::businessDaysBetween()` doesn't count weekends.  Also, `calendar.isBusinessDay(d)` returns false on weekends.  Did you find any cases in which this was not true?

Reply:
I could no longer reproduce the issue I encountered yesterday. Maybe I was chasing ghosts. 
I pushed a commit which makes use of `Calendar::businessDaysBetween()`.